tools/codegen/pkg/schemacheck/generator.go

```diff
@@ -121,6 +122,11 @@ func (g *generator) genGroupVersion(group string, version generation.APIVersionC
 		return fmt.Errorf("could not load schema check generation contexts for group/version %s/%s: %w", group, version.Name, err)
 	}
 
+	if len(contexts) == 0 {
+		klog.V(1).Infof("No CRD manifests found for %s/%s", group, version.Name)
```

Comment:
Can't make this an error as we have genuine cases where there is no CRD, eg user group